src/cheri/attributes.adoc

```diff
@@ -14,6 +14,7 @@
 :cheri_priv_m_dyn_xlen_ext: Smcherivarxlen
 :cheri_priv_s_ext: Sscheri
 :cheri_priv_h_ext: Shcheri
+:cheri_default_cap: Zcheridefaultcap
```

Comment:
Do we actually need this? Can't we say this is the format mandated if you support RV32CH or RV64CH if you don't have another extension (like `zcheriot`) that redefines it?